rites/roman1969/__tests__/brazil-calendar.test.ts

```diff
@@ -7,10 +7,19 @@ import { Ranks } from '../src/constants/ranks';
 describe('Testing Brazilian calendar specific celebrations', () => {
   const romcal = new Romcal({ localizedCalendar: Brazil_PtBr });
 
+  let calendar2024: Awaited<ReturnType<typeof romcal.generateCalendar>>;
+  let calendar2025: Awaited<ReturnType<typeof romcal.generateCalendar>>;
+  let calendar2026: Awaited<ReturnType<typeof romcal.generateCalendar>>;
+
+  beforeAll(async () => {
+    calendar2024 = await romcal.generateCalendar(2024);
+    calendar2025 = await romcal.generateCalendar(2025);
+    calendar2026 = await romcal.generateCalendar(2026);
+  });
+
   describe('Brazilian saints', () => {
-    test('Santa Dulce Lopes Pontes should be celebrated on August 13', async () => {
-      const calendar = await romcal.generateCalendar(2024);
-      const august13 = calendar['2024-08-13'];
+    test('Santa Dulce Lopes Pontes should be celebrated on August 13', () => {
```

Comment:
Could we use English names of the saints please? Right now, you sometimes use Portuguese names, sometimes English names, sometimes with canonisation level and sometimes without it.

I suggest to use `${shortCanoisationLevel} ${nameWithoutTitles}`, e.g. `Sts Peter and Paul`, `St Dulce Lopes Pontes`, `St Joseph de Anchieta`.